### ISSUE TITLE
Are you looking for maintainers?

### ISSUE THREAD

**Issue by ellipticaldoor** [overdrivenpotato/rust-psp#40]
Im looking a project to learn rust and this one seems fun.

There is anything I can help with?

**Comment by overdrivenpotato** [comment]
Yes, definitely. If you (or anybody else reading this) want to help out, feel free to join the `#rust-psp` channel in the PSP Homebrew Community server: https://discord.gg/WY8XhDG
